src/samples/Uno.Material.Samples/Uno.Material.Samples.Shared/SamplesPage.xaml.cs

```diff
@@ -133,22 +135,27 @@ void AddMenuItem<TSamplePage>(string content = null, string icon = null)
 				NavView.MenuItems.Add(new NavigationViewItem()
 				{
 					Content = content ?? Regex.Replace(typeof(TSamplePage).Name, @"SamplePage$", string.Empty),
-					Icon = icon != null
-						? new BitmapIcon()
-						{
-							UriSource = new Uri($"ms-appx:///Assets/NavigationViewIcons/{icon}.png"),
-#if __IOS__
-							// workaround for BitmapIcon not displaying on ios
-							ShowAsMonochrome = false
-#endif
-						}
-						: null,
+					Icon = GenerateMenuItemBitmapIcon(icon),
 					Tag = typeof(TSamplePage),
 				});
 			}
 		}
 
-#endregion
+		private BitmapIcon GenerateMenuItemBitmapIcon(string icon)
+		{
+			var iconName = icon == null ? PlaceholderIcon : icon;
+
+			return new BitmapIcon()
+			{
+				UriSource = new Uri($"ms-appx:///Assets/NavigationViewIcons/{iconName}.png"),
+#if __IOS__
+				// workaround for BitmapIcon not displaying on ios
+				ShowAsMonochrome = false
+#endif
+			};
```

Comment:
![Annotation 2020-09-09 093536](https://user-images.githubusercontent.com/17461593/92605607-eda4f280-f27f-11ea-8052-9eb95ff4c7a6.jpg)

Should we also the placeholder icon for the controlextension item ?